### ISSUE TITLE
prov: hadActivity

### ISSUE THREAD

**Issue by giacomodecolle** [BFO-Mappings/BFO-Mappings#21]
It looks like this should be a relation between an Influence and Activity that "has this activity". We are trying to understand whether this is a parthood relation. 

"The _optional_ Activity of an Influence, which used, generated, invalidated, or was the responsibility of some Entity. This property is _not_ used by ActivityInfluence (use prov:activity instead)."

Austin raised the issue that if, say, delegation is a parthood relation, and :hadActivity ranges over two processes that occur one after another, then it seems it doesn't make sense.
Example: I delegate to you to do x. Is the prov:hadActivity property ranging between my Influence of delegating and the process of delegating, or does it range between your Activity of executing it and my Influence of delegating? Only in the first case this would be a parthood relation. 

Basically, we need to find some examples of use of :hadActivity in the documentation.

@tmprd @avsculley @SydCo99 @tmprd

**Comment by tmprd** [comment]
I think it's the latter: "range between your Activity of executing it and my Influence of delegating".
https://www.w3.org/TR/prov-o/#qualifiedDerivation
```turtle
:bar_chart
   a prov:Entity;
   prov:wasDerivedFrom :aggregatedByRegions;
   prov:qualifiedDerivation [
      a prov:Derivation; # a prov:Influence
      prov:entity :aggregatedByRegions; 
      
      ## More details about the activity underpinning the derivation        
      prov:hadGeneration :chat_plotting; 
      prov:hadActivity   :chart_creation ;
   ];
```
So a bar chart was derived from some data. That derivation was a kind of influence and it's related to ("hadActivity") the activity of creating a bar chart. With our current BFO mapping, this entails that this derivation was a temporal part of the activity of creating a bar chart. That makes sense to me.

Current BFO mapping
```turtle
[]  rdf:type owl:Axiom ;
    owl:annotatedSource   prov:hadActivity ;
    owl:annotatedProperty rdfs:subPropertyOf ;
    owl:annotatedTarget   obo:BFO_0000139 ;
    sssom:object_label    "temporal part of" ;
    sssom:mapping_justification "A prov Delegation or Derivation or End or Start that has activity some prov:Activity is also a temporal part of that prov:Activity."@en .
```

Here's an example of the kind you mentioned with delegation: https://www.w3.org/TR/prov-o/#Delegation
```turtle
:insuranceAgent_Frank
   a prov:Person;
   prov:actedOnBehalfOf :insuranceCompany_A;
   prov:qualifiedDelegation [
      a prov:Delegation;
      prov:agent        :insuranceCompany_A;
      ex:rewardScheme   "commission";
      prov:hadActivity  :policySale ;
   ];
```
The activity of selling an insurance policy is related to the delegation (of this activity?) to an insurance agent by an insurance company. Was that delegation a temporal part of the activity of selling the insurance policy? I agree that this doesn't work. 

We could change `rdfs:subPropertyOf` to `skos:broaderMatch` ... the mapping works logically or "structurally" in that the domains and ranges fit.

**Comment by tmprd** [comment]
Using RO "causal relation between processes"
